footer.js

```diff
@@ -24,6 +24,10 @@ module.exports.footer = {
           label: 'Integrations',
           to: '/integrations',
         },
+        {
+          label: 'Chrome plugin',
+          to: 'https://chrome.google.com/webstore/detail/tolgee-tools/hacnbapajkkfohnonhbmegojnddagfnj',
```

Comment:
```suggestion
          href: 'https://chrome.google.com/webstore/detail/tolgee-tools/hacnbapajkkfohnonhbmegojnddagfnj',
```
It should be `href` as it is pointing to a different domain (same as other links, e.g. to Slack, GitHub,...)

Reply:
Ok